packages/testing/src/api/index.ts

```diff
@@ -128,24 +136,70 @@ export type LoadDatabaseRulesOptions = {
   databaseName: string;
   rules: string;
 };
-export function loadDatabaseRules(options: LoadDatabaseRulesOptions): void {
+export function loadDatabaseRules(
+  options: LoadDatabaseRulesOptions
+): Promise<void> {
   if (!options.databaseName) {
-    throw new Error('databaseName not specified');
+    throw Error('databaseName not specified');
+  }
+
+  if (!options.rules) {
+    throw Error('must provide rules to loadDatabaseRules');
+  }
+
+  return new Promise((resolve, reject) => {
+    request.put(
+      {
+        uri: `http://${DATABASE_ADDRESS}/.settings/rules.json?ns=${
+          options.databaseName
+        }`,
+        headers: { Authorization: 'Bearer owner' },
+        body: options.rules
+      },
+      (err, resp, body) => {
+        if (err) {
+          reject(err);
+        } else {
+          resolve();
+        }
+      }
+    );
+  });
+}
+
+export type LoadFirestoreRulesOptions = {
+  projectId: string;
+  rules: string;
+};
+export function loadFirestoreRules(
+  options: LoadFirestoreRulesOptions
+): Promise<void> {
+  if (!options.projectId) {
+    throw new Error('projectId not specified');
   }
 
   if (!options.rules) {
-    throw new Error('must provide rules to loadDatabaseRules');
+    throw new Error('must provide rules to loadFirestoreRules');
   }
 
-  request({
-    uri: `http://${DATABASE_ADDRESS}/.settings/rules.json?ns=${
-      options.databaseName
-    }`,
-    method: 'PUT',
-    headers: { Authorization: 'Bearer owner' },
-    body: options.rules
-  }).catch(function(err) {
-    throw new Error('could not load rules: ' + err);
+  let client = new EMULATOR.FirestoreEmulator(
+    'localhost:8080',
```

Comment:
we allow the customer to change the port that the emulator runs on

Reply:
Good point, changed this to `FIRESTORE_ADDRESS`